### PR TITLE
fix: Avoid error where we try to compute refunds for a {l1Token, chainId} combination that doesn't exist

### PR DESCRIPTION
**Pull request by nicholaspai**
For example, when `getBundleRefunds()` is called with `l1Token = USDT address`, then this function fails because USDT doesn't exist on Base, which is one of the enabled chains.

### CODE REVIEW

**Review by pxrl — APPROVED** [review]
Really nice catch!

File: src/clients/InventoryClient.ts [hunk 1]
```diff
@@ -166,13 +166,15 @@ export class InventoryClient {
...
-      this.getEnabledChains().map((chainId) => {
-        const destinationToken = this.getDestinationTokenForL1Token(l1Token, chainId);
-        return [
-          chainId,
-          this.bundleDataClient.getTotalRefund(refundsToConsider, this.relayer, Number(chainId), destinationToken),
-        ];
-      })
+      this.getEnabledChains()
+        .filter((chainId) => this.hubPoolClient.l2TokenEnabledForL1Token(l1Token, chainId))
+        .map((chainId) => {
```
Comment (james-a-morris) [inline src/clients/InventoryClient.ts:171]: Great catch

**Review by james-a-morris — APPROVED** [review]
+1 - thoughts on using a reduce to prevent the double loop?

**Comment by nicholaspai** [comment]
> +1 - thoughts on using a reduce to prevent the double loop?

yeah good call

**Pull request merged**